src/meshapi/tests/test_join_form.py

```diff
@@ -226,34 +233,118 @@ def test_bad_address_join_form(self):
     def test_member_moved_join_form(self):
         # Name, email, phone, location, apt, rooftop, referral
         form, s = pull_apart_join_form_submission(valid_join_form_submission)
-        response = self.c.post("/api/v1/join/", form, content_type="application/json")
+        response1 = self.c.post("/api/v1/join/", form, content_type="application/json")
 
         code = 201
         self.assertEqual(
             code,
-            response.status_code,
-            f"status code incorrect for Valid Join Form. Should be {code}, but got {response.status_code}.\n Response is: {response.content.decode('utf-8')}",
+            response1.status_code,
+            f"status code incorrect for Valid Join Form. Should be {code}, but got {response1.status_code}.\n Response is: {response1.content.decode('utf-8')}",
         )
 
-        validate_successful_join_form_submission(self, "Valid Join Form", s, response)
+        validate_successful_join_form_submission(self, "Valid Join Form", s, response1)
 
-        # Now test that the member can "move" and still access the jon form
+        # Now test that the member can "move" and still access the join form
         v_sub_2 = valid_join_form_submission.copy()
         v_sub_2["street_address"] = "152 Broome Street"
 
         form, s = pull_apart_join_form_submission(v_sub_2)
 
         # Name, email, phone, location, apt, rooftop, referral
-        response = self.c.post("/api/v1/join/", form, content_type="application/json")
+        response2 = self.c.post("/api/v1/join/", form, content_type="application/json")
 
         code = 201
         self.assertEqual(
             code,
-            response.status_code,
-            f"status code incorrect for Valid Join Form. Should be {code}, but got {response.status_code}.\n Response is: {response.content.decode('utf-8')}",
+            response2.status_code,
+            f"status code incorrect for Valid Join Form. Should be {code}, but got {response2.status_code}.\n Response is: {response2.content.decode('utf-8')}",
+        )
+
+        validate_successful_join_form_submission(self, "Valid Join Form", s, response2)
+
+        self.assertEqual(
+            json.loads(
+                response1.content.decode("utf-8"),
+            )["member_id"],
+            json.loads(
+                response2.content.decode("utf-8"),
+            )["member_id"],
         )
 
-        validate_successful_join_form_submission(self, "Valid Join Form", s, response)
+    def test_member_moved_and_used_stripe_email_join_form(self):
```

Comment:
damn nice edge case

Reply:
<3

Needed to get coverage on my new feature somehow